backends/prometheus/README.md

```diff
@@ -22,15 +24,16 @@ http://your.netdata.ip:19999
 *(replace `your.netdata.ip` with the IP or hostname of the server running netdata)*
 
 ### Installing Prometheus
+
 In order to install prometheus we are going to introduce our own systemd startup script along with an example of prometheus.yaml configuration. Prometheus needs to be pointed to your server at a specific target url for it to scrape netdata's api. Prometheus is always a pull model meaning netdata is the passive client within this architecture. Prometheus always initiates the connection with netdata.
 
-##### Download Prometheus
+#### Download Prometheus
 
 ```sh
 wget -O /tmp/prometheus-2.3.2.linux-amd64.tar.gz https://github.com/prometheus/prometheus/releases/download/v2.3.2/prometheus-2.3.2.linux-amd64.tar.gz
```

Comment:
Why we are referencing locked down version of software we don't control? This means our docs will get stale pretty fast (prometheus is already at 2.4.x and in next days there will be 2.5.0 release)

Reply:
Your concern is valid, however it's out of scope for this PR, I just did some formatting changes. I suggest you open an issue instead so that this important topic will be discussed (and hopefully resolved).